### PR TITLE
Adding Mac ARM64 functionality to shield

### PR DESCRIPTION
**Pull request by antitux**
This adds functionality to shield to build native arm64 binaries and updates the shield UI to point to both intel and arm64 binaries.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -125,7 +127,8 @@ release:
 	cd artifacts && for x in shield-server-*; do \
 	  cp -a ../web/htdocs $$x/webui; \
 	  mkdir -p $$x/webui/cli/linux; cp ../artifacts/shield-linux-amd64   $$x/webui/cli/linux/shield; \
-	  mkdir -p $$x/webui/cli/mac;   cp ../artifacts/shield-darwin-amd64  $$x/webui/cli/mac/shield; \
+	  mkdir -p $$x/webui/cli/mac-amd64;   cp ../artifacts/shield-darwin-amd64  $$x/webui/cli/mac-amd64/shield; \
+	  mkdir -p $$x/webui/cli/mac-arm64; cp ../artifacts/shield-darwin-arm64 $$x/webui/cli/mac-arm64/shield; \
```
Comment (wayneeseguin) [inline Makefile:131]: Perhaps it would be cleaner to do mac/{amd64,arm64} as subdirectories?

Comment (wayneeseguin) [reply Makefile:131]: Additionally I think we should stick to uname output and have it be `darwin/$ARCH`

Comment (wayneeseguin) [reply Makefile:131]: More clear statement: it should be `darwin/{amd64/arm64}` not `mac-*`

Comment (antitux) [reply Makefile:131]: I'm fine with that too.

**Pull request merged**